### PR TITLE
Add placeholder props to support multi locales

### PR DESCRIPTION
**Pull request by cmpsleo**

### CODE REVIEW

**Comment by albingroen** [comment]
Looks good!

**Review by albingroen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -194,6 +195,15 @@ useEffect(() => {
 
 The list item also extends the `HTMLAnchorElement & HTMLButtonElement` types
 
+### `CommandPalette.FreeSearchAction`
+
+| name           | type                     | required | default    | description                                |
+| -------------- | ------------------------ | -------- | ---------- | -------------------------------------------|
+| index          | number                   | false    | `0`        | Index for list item                        |
+| placeholder    | string                   | false    | `"Search"` | Button placeholder                         |
```
Comment (albingroen) [inline README.md:203]: I think the default value here should be `"Search for"`, right?

**Review by albingroen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (albingroen) [reply README.md:203]: I also think the prop should be called something else than placeholder. Probably `prefix`.

Comment (cmpsleo) [reply README.md:203]: What do you think about `label`?

Comment (albingroen) [reply README.md:203]: Label is fine by me

Comment (cmpsleo) [reply README.md:242]: Okay, I will change to it.

**Comment by albingroen** [comment]
Seems to be a conflict in the README file. Please try rebasing locally and fixing the conflict 👍🏼

**Comment by cmpsleo** [comment]
> Seems to be a conflict in the README file. Please try rebasing locally and fixing the conflict 👍🏼

It's all updated for me. What specific part of `README` do you talk about?

**Comment by cmpsleo** [comment]
> Thanks for fixing the feedback! Feel free to go ahead and merge and I'll update the version and publish it later 👍🏼

I don't have `write access` to do it, could you give it to me? I think that it's more clear for me to resolve conflicts too because I can visualize the file comparison.

**Comment by albingroen** [comment]
> > Thanks for fixing the feedback! Feel free to go ahead and merge and I'll update the version and publish it later 👍🏼
> 
> I don't have `write access` to do it, could you give it to me? I think that it's more clear for me to resolve conflicts too because I can visualize the file comparison.

I can merge it when the conflicts are resolved. I cloned your repo and tried rebasing and fixing the conflicts and it seemed to work, so it should for you too.

**Comment by cmpsleo** [comment]
I'm so sorry about the lateness, but It's my first open-source contribution 😬

**Comment by albingroen** [comment]
> I'm so sorry about taking this too long, but It's my first open-source contribution 😬

No worries mate - great job! One last thing, it would be great if you could squash these 3 commits together and remove the prefix (like `"feat:"`).

**Comment by cmpsleo** [comment]
> > I'm so sorry about taking this too long, but It's my first open-source contribution 😬
> 
> No worries mate - great job! One last thing, it would be great if you could squash these 3 commits together and remove the prefix (like `"feat:"`).

Done :)

**Pull request merged**